psl-evaluation/src/main/java/org/linqs/psl/utils/evaluation/statistics/DiscretePredictionComparator.java

```diff
@@ -78,7 +78,7 @@ public void setResultFilter(AtomFilter af) {
 	@Override
```

Comment:
Since we are already making changes in DiscretePredictionComparator, lets replace the iterators on lines 96 and 135 with a foreach loop. Also, change the if on line 144 to a negative condition and continue if it gets in. (In general, we want to reduce the complexity of the code and doing things like this lets us reduce the indentation level and mental burden.